modules/service-autoscaling/variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "cluster_name" {
+  description = "ECS cluster name. Uses default cluster if not supplied."
+  default     = ""
+}
+
+variable "service_name" {
+  description = "ECS service name"
+}
+
+variable "max_capacity" {
+  description = "Maximum number of tasks that the autoscaling policy can set for the service."
+}
+
+variable "min_capacity" {
+  description = "Minimum number of tasks that the autoscaling policy can set for the service."
+  default     = 0
+}
+
+variable "queue_name" {
+  description = "Name of the queue to compute QueueBacklog metric for."
+}
+
+variable "target_value" {
+  description = "Queue backlog (in seconds) to maintain for the service when under maximum load."
```

Comment:
Is there a good reasonable default to set this to?

Reply:
Not in my opinion. It's too dependent on the service. There's probably reasonable upper and lower limits (in most cases), but the range within those is too large to really give a default.

Reply:
I think it's probably worth setting something insanely conservative as the default, mostly to simplify the ability to get it up and running as soon as possible.

Reply:
Ok. Done.